### PR TITLE
Presentation w1

### LINKED ISSUE #11: Create the presentation 1
Blocked by #3

Presentation 1 presents the team, describes the project that we will be building, its features, the tech stack, our ambitions regarding Quality, Reliability and Security. Should also mention what we've done over these two weeks.

**Acceptance criteria**:
- `pnpm dev` in the `presentation` project opens the presentation
- Slides have presenter notes to highlight key points of what to talk about during this slide

### PR DESCRIPTION
**Pull request by alkaitagi**
Adds slides for week-1 presentation. Closes #11.

### CODE REVIEW

**Review by illright — APPROVED** [review]
Lovely

File: presentation/slides.md [hunk 1]
```diff
@@ -1,18 +1,99 @@
...
+- 📱 Use on any device with browser (**PWA**)
+
+<p class="absolute bottom-10 text-sm">Cast • 2</p>
+
+---
+layout: image-right
+---
+Sprint #0: Progress
+
+- 🫂 Team building & setting up collaboration
```
Comment (aabounegm) [inline presentation/slides.md:51]: The icon here appears as a box with a question mark for me. Could it be some emoji added only in Windows 11?

File: presentation/slides.md [hunk 2]
```diff
@@ -1,18 +1,99 @@
...
+- 💻 Initialization of repozitory, environment
```
Comment (aabounegm) [inline presentation/slides.md:52]: repo**s**itory (and no, not even in American English :P)

File: presentation/slides.md [hunk 3]
```diff
@@ -1,18 +1,99 @@
...
+<p class="absolute bottom-10 text-sm">Cast • 5</p>
+
+---
+layout: image-right
+image: "img/mike-hindle-urTBnMtWWYc-unsplash.jpg"
+---
+
+# Course Goals #2
+
+## Opreations
```
Comment (aabounegm) [inline presentation/slides.md:92]: Op**er**ations

**Review by aabounegm — COMMENTED** [review]
Just a couple of typos

File: presentation/slides.md (hunk 2 above)
Comment (alkaitagi) [reply presentation/slides.md:52]: Dagestani Inglish

File: presentation/slides.md (hunk 1 above)
Comment (alkaitagi) [reply presentation/slides.md:51]: Possibly.

**Pull request merged**